cmd/common/registry.go

```diff
@@ -138,7 +150,9 @@ func buildCommandFromGadget(
 
 	// Get per gadget operator params
 	validOperators := operators.GetOperatorsForGadget(gadgetDesc)
-	operatorsParamsCollection := validOperators.ParamCollection()
+
+	// TODO: Combine remote operator params with locally available ones
+	//  Example use case: setting default namespace for kubernetes
```

Comment:
This is interesting. As you told me privately, I think having a local version of the kubemanager operator seems a possibility. I have tried other approaches but all of them put operator-specific logic on registry.go and other places.